test/e2e/operator.go

```diff
@@ -379,33 +379,79 @@ var _ = Describe("ARO Operator - Azure Subnet Reconciler", func() {
 })
 
 var _ = Describe("ARO Operator - MUO Deployment", func() {
-	ctx := context.Background()
+	const (
+		managedUpgradeOperatorNamespace = "openshift-managed-upgrade-operator"
+		interval                        = 30 * time.Second
+		duration                        = 10 * time.Minute
+	)
+
+	var ctx context.Context
+
+	muoIsDeployed := func() (bool, error) {
+		By("getting MUO pods")
+
+		listOptions := metav1.ListOptions{
+			LabelSelector: "name=managed-upgrade-operator",
+		}
+		pods, err := clients.Kubernetes.CoreV1().Pods(managedUpgradeOperatorNamespace).List(ctx, listOptions)
+		if err != nil {
+			return false, err
+		}
+		if len(pods.Items) != 1 {
+			return false, fmt.Errorf("%d managed-upgrade-operator pods found", len(pods.Items))
+		}
+
+		By("getting logs from MUO")
+		b, err := clients.Kubernetes.CoreV1().Pods(managedUpgradeOperatorNamespace).GetLogs(pods.Items[0].Name, &corev1.PodLogOptions{}).DoRaw(ctx)
+		if err != nil {
+			return false, err
+		}
+
+		By("verifying that MUO has FIPS crypto mandated by reading logs")
+		return strings.Contains(string(b), `msg="FIPS crypto mandated: true"`), nil
+	}
+
+	BeforeEach(func() {
+		ctx = context.Background()
```

Comment:
is there any benefit to just setting it to `context.Background()` once?

Reply:
Done.